supertokens_python/recipe/session/interfaces.py

```diff
@@ -268,7 +260,7 @@ async def remove_claim(
         self,
         session_handle: str,
         claim: SessionClaim[Any],
-        user_context: Dict[str, Any],
+        user_context: Optional[Dict[str, Any]] = None,
```

Comment:
userContext here should not be optional since it's part of recipe interface

Reply:
done